Content.Shared/NPC/Components/NpcFactionMemberComponent.cs

```diff
@@ -2,10 +2,11 @@
 using Content.Shared.NPC.Systems;
 using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
+using Content.Shared.Mech.EntitySystems; // Frontier
 
 namespace Content.Shared.NPC.Components;
 
-[RegisterComponent, NetworkedComponent, Access(typeof(NpcFactionSystem))]
+[RegisterComponent, NetworkedComponent, Access(typeof(NpcFactionSystem), typeof(SharedMechSystem))] // Frontier - Added MechSystem
```

Comment:
Does this even need an access component?  I can't find any other partial implementations, and all of the fields here are public.  Am I missing something?

Reply:
"Read Write only" error
I did need it to set factions

Reply:
Ah, gotcha.  Thank you.